packages/arcs/src/arc_labels/ArcLabel.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { CSSProperties } from 'react'
```

Comment:
Not a big fan of the underscore in the folder name, how about a dash (`-`)?

Reply:
I prefer to use underscores :) I guess it's a matter of habits, using python lately, underscores in file names are one.
I don't think it makes a huge difference in the end, but I would prefer to stick with this.